amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/common/transformation/stage/ParametersNormalizationStage.scala

```diff
@@ -25,34 +26,58 @@ abstract class ParametersNormalizationStage(profile: ProfileName) extends Transf
                          configuration: AMFGraphConfiguration): BaseUnit = model match {
     case doc: Document if doc.encodes.isInstanceOf[Api] =>
       val api = doc.encodes.asInstanceOf[Api]
-      transform(api)
+      transform(api)(errorHandler)
       doc
     case _ => model
   }
 
-  protected def transform(api: Api): Api = api
+  protected def transform(api: Api)(implicit errorHandler: AMFErrorHandler): Api = api
 
-  protected def pushParamsToEndpointOperations(endpoint: EndPoint, finalParams: Parameters) = {
+  protected def pushParamsToEndpointOperations(endpoint: EndPoint, finalParams: Parameters)(
+      implicit errorHandler: AMFErrorHandler) = {
     endpoint.operations.foreach { op =>
-      setRequestParameters(op, finalParams)
+      setRequestParameters(op, finalParams)(errorHandler)
```

Comment:
`Implicit` parameters don't have to be propagated to every method. It is one of the benefits of using `implicit` variables. This happens quite a lot in this file.